tests/unit/client.py

```diff
@@ -837,6 +837,11 @@ def test_network_list(self):
                 u'net-05': {u'network_id': u'1005', u'projects': [u'proj-02']}
                 }
 
+    def test_list_network_attachments(self):
```

Comment:
Can we have a test for when the result is non empty?

Reply:
added!